src/PROPOSAL/Secondaries.cxx

```diff
@@ -394,12 +395,11 @@ std::vector<ContinuousLoss> Secondaries::GetContinuousLosses() const
     std::vector<ContinuousLoss> losses;
     for (unsigned int i=1; i<track_.size(); i++) {
         if (types_[i] == InteractionType::ContinuousEnergyLoss) {
-            losses.emplace_back(
-                    std::make_pair(track_[i-1].energy, track_[i].energy),
-                    std::make_pair(track_[i-1].position, track_[i].position),
-                    std::make_pair(track_[i-1].direction, track_[i].direction),
-                    std::make_pair(track_[i-1].time, track_[i].time)
-                    );
+            losses.emplace_back( track_[i].energy - track_[i-1].energy,
```

Comment:
this change should be mentioned in the description since it it is not directly related to this PR topic